src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -602,13 +602,15 @@ private static Properties loadProperties(File file)
     }
 
     /**
-     * Creates the audit listener.
+     * Creates the audit listener. The JDT compiler issues the warning explicitlyClosedAutoCloseable
+     * for {@code Files.newOutputStream}, so it need to be suppressed.
```

Comment:
this javadoc is public, let be more accurate here.

please remove "The JDT compiler issues the warning explicitlyClosedAutoCloseable ...." and make it like `This method create in AuditListener open stream for validation data, it must be closed by {@link RootModule} (default implementation is {@link Checker}) by launching {@link AuditListener#auditFinished(AuditEvent)}`